virtool/references/data.py

```diff
@@ -982,3 +985,16 @@ async def clean_all(self):
                         {"$pop": {"updates": -1}, "$set": {"updating": False}},
                         session=session,
                     )
+
+    async def update_references(self):
+        while True:
```

Comment:
Scheduling + repetition will be handled by the task spawner. Task should run only once per call, otherwise it will block other tasks for running